client/test/src/main/java/org/elasticsearch/client/RestClientTestUtil.java

```diff
@@ -81,4 +85,22 @@ static int randomErrorRetryStatusCode(Random random) {
     static List<Integer> getAllStatusCodes() {
         return ALL_STATUS_CODES;
     }
+
+    /**
+     * Create the specified number of {@link Header}s.
```

Comment:
Do we want to be able to specify the number as the "generate" method was able before? If not the comment should be changed.